File: pkg/doctor/testdata-packs/tax-percent/positive.php
<?php
$stripe = new \Stripe\StripeClient('sk_test_x');

$stripe->subscriptions->update('sub_123', [
  'tax_percent' => 19.0,
]);

// legacy static-call SDK style
\Stripe\Invoice::create([
  'customer' => 'cus_123',
  'tax_percent' => 10.0,
]);

File: pkg/doctor/testdata-packs/tax-percent/positive.rb
require 'stripe'

Stripe::Subscription.create(
  customer: 'cus_123',
  items: [{price: 'price_123'}],
  tax_percent: 20.0,
)

Stripe::Invoice.update(
  'in_123',
  tax_percent: 7.5,
)
